app/bin/setup_kinto.py

```diff
@@ -31,51 +32,52 @@ def create_user(user, passw):
 print(f">>>> Creating kinto bucket: {os.environ['KINTO_BUCKET']}")
 print(
     client.create_bucket(
-        id=os.environ["KINTO_BUCKET"],
+        id=KINTO_BUCKET,
         permissions={"read": ["system.Everyone"]},
         if_not_exists=True,
     )
 )
 
 
-print(">>>> Creating kinto group: editors")
-print(
-    client.create_group(
-        id=f"{os.environ['KINTO_COLLECTION']}-editors",
-        bucket=os.environ["KINTO_BUCKET"],
-        data={"members": [f"account:{os.environ['KINTO_USER']}"]},
-        if_not_exists=True,
+for collection in os.environ["KINTO_COLLECTION"], os.environ["KINTO_COLLECTION_NIMBUS"]:
```

Comment:
How come these aren't pulled out at the top? Usually when I write CLI type code like this in Python I use the `click` package, so that there's useful help for the script as well as argument checking up front.